tests/modules/manta/germline/main.nf

```diff
@@ -7,29 +7,46 @@ include { MANTA_GERMLINE } from '../../../../modules/manta/germline/main.nf'
 workflow test_manta_germline {
     input = [
         [ id:'test'], // meta map
-        file(params.test_data['homo_sapiens']['illumina']['test_paired_end_recalibrated_sorted_cram'], checkIfExists: true),
-        file(params.test_data['homo_sapiens']['illumina']['test_paired_end_recalibrated_sorted_cram_crai'], checkIfExists: true)
+        [ file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_cram'], checkIfExists: true)],
```

Comment:
what is the reason that you switched the test files?

Reply:
Ah, because there was mismatch between the fasta header and the reference sequence in the cram. The genome fasta sequence is from chr22, and cram alignments are from chr21. Hence, the switch.